model/distro/distro.go

```diff
@@ -47,6 +47,7 @@ type Distro struct {
 	HostAllocatorSettings HostAllocatorSettings `bson:"host_allocator_settings" json:"host_allocator_settings" mapstructure:"host_allocator_settings"`
 	DisableShallowClone   bool                  `bson:"disable_shallow_clone" json:"disable_shallow_clone" mapstructure:"disable_shallow_clone"`
 	Note                  string                `bson:"note" json:"note" mapstructure:"note"`
+	Warnings              string                `bson:"warnings,omitempty" json:"warnings,omitempty" mapstructure:"warnings,omitempty"`
```

Comment:
If this can be multiple warnings should it be a list instead? If not let's rename it to something that doesn't imply there being multiple potential warnings

Reply:
I updated it with Arjuns suggestion!